src/odemis/acq/stream/_sync.py

```diff
@@ -812,25 +823,45 @@ def _estimateRawAcquisitionTime(self):
             res = self._sccd._getDetectorVA("resolution").value
             readout = numpy.prod(res) / ro_rate
 
-            exp = self._sccd._getDetectorVA("exposureTime").value
+            if hasattr(self._sccd, "integrationTime"):
+                exp = self._sccd.integrationTime.value
+                # calculate number of images to integrate
+                hwExpTimeMax = self._sccd.detector.exposureTime.range[1]
+                numImages2integrate = int(numpy.ceil(exp / hwExpTimeMax))
+                readout *= numImages2integrate
+            else:
+                exp = self._sccd._getDetectorVA("exposureTime").value
+
             dur_image = (exp + readout + 0.03) * 1.20
             duration = numpy.prod(self.repetition.value) * dur_image
             # Add the setup time
             duration += self.SETUP_OVERHEAD
 
             return duration
+
         except Exception:
             msg = "Exception while estimating acquisition time of %s"
             logging.exception(msg, self.name.value)
+
             return Stream.estimateAcquisitionTime(self)
 
     def _adjustHardwareSettings(self):
         """
-        Read the SEM and CCD stream settings and adapt the SEM scanner
-        accordingly.
-        return (float): estimated time for a whole CCD image
+        Read the SEM and CCD stream settings and adapt the SEM scanner accordingly.
+        :returns: exp + readout (float): Estimated time for a whole (but not integrated) CCD image.
         """
-        exp = self._sccd._getDetectorVA("exposureTime").value  # s
+        if model.hasVA(self._sccd, "integrationTime"):
+            # calculate exposure time to be set on detector
+            exp = self._sccd.integrationTime.value / self._sccd.integrationCounts.value
+            # TODO should we have a try/except here in case of value out of range
+            #  or should that already be handled somewhere else...?
+            self._sccd.detector.exposureTime.value = exp
```

Comment:
Actually, the hardware may pick a different value (for instance, because it always rounds the value to 0.1s). So you might want to play it super safe, which such (seemingly redundant) code:
```
exp = self._sccd.integrationTime.value / self._sccd.integrationCounts.value
self._ccd.exposureTime.value = self._ccd.exposureTime.clip(exp)
integ_count = int(math.ceil(self._sccd.integrationTime.value / self._ccd.exposureTime.value))
```
(and maybe add an extra log if `integ_count != self._sccd.integrationCounts.value`.)